packages/graph/hash_graph/lib/graph/src/api/rest/entity.rs

```diff
@@ -125,6 +134,23 @@ async fn get_entity<P: GraphPool>(
         .map(Json)
 }
 
+#[utoipa::path(
+get,
+path = "/entities",
+tag = "Entity",
+responses(
+(status = 200, content_type = "application/json", description = "List of all entities", body = [Entity]),
+(status = 500, description = "Store error occurred"),
+)
+)]
```

Comment:
rustfmt is not able to format this attribute, could you adjust that please?

Reply:
Done